medical_prescription_merge/wizards/medical_prescription_order_merge.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import itertools
+from openerp import _, api, fields, models
+from openerp.exceptions import ValidationError
+
+
+class MedicalPrescriptionOrderMerge(models.TransientModel):
+    _name = 'medical.prescription.order.merge'
+
+    merge_order_ids = fields.Many2many(
+        comodel_name='medical.prescription.order',
+        string='Orders to be Merged',
+        help='List of prescription orders to be merged',
+        required=True,
+        default=lambda s: s._compute_default_merge_orders(),
+    )
+    dest_order_id = fields.Many2one(
+        comodel_name='medical.prescription.order',
+        string='Destination Order',
+        help='This prescription order will serve as the baseline for the merge'
+             ' (its info will take precedence when there are conflicts)',
+        required=True,
+        default=lambda s: s._compute_default_dest_order(),
+        # domain='[("id", "in", merge_order_ids[0][2])]',
```

Comment:
@lasley - As per our convo earlier, the wizard should only allow users to select a destination order that is in the merge order list. The domain shown here will do that, but you decided that it was too hacky and wanted to look into it further as part of the PR review.